core/src/main/java/com/infinum/jsonapix/core/resources/DefaultError.kt

```diff
@@ -6,8 +6,8 @@ import kotlinx.serialization.Serializable
 @Serializable
 @SerialName("error")
 class DefaultError(
-    @SerialName("code") val code: String,
-    @SerialName("title") val title: String,
-    @SerialName("detail") val detail: String,
-    @SerialName("status") val status: String
+    @SerialName("code") val code: String? = null,
+    @SerialName("title") val title: String? = null,
+    @SerialName("detail") val detail: String? = null,
+    @SerialName("status") val status: String? = null,
```

Comment:
After reviewing JsonApi standards, default errors can have any or all of these attrs, in case the developer wants to make it not null, they will have to create a custom error class for that.

and for developers that want to have one type of errors for multiple classes, this will also be achievable using this [repeatable annotations support ](https://app.productive.io/1-infinum/tasks/6964170) or create a [custom default error](https://app.productive.io/1-infinum/tasks/7263508) class for every other class

Reply:
What do you mean by having one type of error for multiple classes and using repeatable annotations? How would it work 🤔 I know currently we would have to specify a custom error for each class right?